docker/nginx/conf.d/include/track-upload

```diff
@@ -13,14 +13,35 @@ log_by_lua_block {
             method = "POST",
             headers = { ["Cookie"] = "skynet-jwt=" .. jwt },
         })
-        
+
         if err or (res and res.status ~= ngx.HTTP_NO_CONTENT) then
             ngx.log(ngx.ERR, "Failed accounts service request /track/upload/" .. skylink .. ": ", err or ("[HTTP " .. res.status .. "] " .. res.body))
         end
     end
 
+    local function scan(premature, skylink, jwt)
```

Comment:
should we extract this function into a separate file and `include` it? That way we have a single place to update when we want to alter the malware scanner ip

Reply:
I agree that it's a good idea. I just wanted to make minimal changes there because I am really uncomfortable in nginx. I'll let @kwypchlo make all decisions on that front. :)

Reply:
I'll unresolve this so it does not go unnoticed